docs/application/native/guides/multimedia/media-conversions.md

```diff
@@ -1,17 +1,9 @@
 # Media Conversions
 
 
-You can perform various media conversions through codecs. You can encode and decode video and audio data, and, in mobile applications, you can also use video transcoding features, such as multi-format codec and output format.
+You can perform various media conversions through codecs. You can encode and decode video and audio data.
```

Comment:
Global: When I had a quick look at the file, I see at many places the word "function" is used. For example,

Set the codec configuration with the mediacodec_set_codec(), and mediacodec_set_vdec_info() or mediacodec_set_venc_info() functions.

It is recommended not to use the word "function" as it is understood. 

please remove the word function, for example, modify the sentence to:

Set the codec configuration with the mediacodec_set_codec(), and mediacodec_set_vdec_info() or mediacodec_set_venc_info().

Reply:
Hello, @surbhig12.
As this comment is minor issue, so I think we can skip the comment,
so that @neoiomok can only focus on the other comment only.

@neoiomok, please skip this comment.